doc/faq.rst

```diff
@@ -0,0 +1,166 @@
+
+|hr|
+
+.. _q_multiple_models:
+
+:fa:`quora`. **Can I pass a set of learners to the** ``model`` **option in the
+configuration file or do I need a separate run for each leaner I want to try?**
+[:ref:`# <q_multiple_models>`]
+
+``rsmtool`` cannot train multiple models via a configuration file. If you wish
+to use multiple learners/models, you should use the :ref:`RSMTool API <api>`
+instead of the command line.
+
+|hr|
+
+.. _q_how_to_predict_with_model:
+
+:fa:`quora`. **Can I compute predictions on new data using a model trained
+with** ``rsmtool`` **?** [:ref:`# <q_how_to_predict_with_model>`]
+
+Yes! We have built :ref:`rsmpredict<usage_rsmpredict>` to do exactly this!
+
+|hr|
+
+.. _q_changed_feature_signs:
+
+:fa:`quora`. **Why did** ``rsmtool`` **change the sign of some features in**
+``feature.csv`` **? I thought** ``rsmtool`` **assumes a positive sign for all
+features?** [:ref:`# <q_changed_feature_signs>`]
+
+``rsmtool`` indeed assumes a default positive sign for all raw features.
+However, if you set ``select_transformations`` to ``true`` in your experiment
+configuration file,  RSMTool will automatically apply
+:ref:`transformations<select_transformations_rsmtool>` to some of the features.
+Some transformations such as ``inv`` (inverse transform) change the polarity of
+the feature.  In such cases, RSMTool takes this into account and changes the
+sign accordingly. See :ref:`note here<clever_sign_note>`.
+
+|hr|
+
+.. _q_mismatched_rsmpredict_rsmtool:
+
+:fa:`quora`. I ran ``rsmpredict`` to generate predicted scores using a model
+trained with ``rsmtool``. I used the evaluation set from ``rsmtool`` as the input
+data for ``rsmpredict``. I expected the number of excluded/included responses
+to be the same. However, for some reason, ``rsmpredict`` generated the
+predicted scores for more cases than ``rsmtool``.
+[:ref:`# <q_mismatched_rsmpredict_rsmtool>`]
+
+The situation described here often occurs in the following situation.
+
+1. ``rsmtool`` is run with one of the built-in :ref:`feature selection
+   models<automatic_feature_selection_models>` and excludes responses where
+   *any* of the candidate features has missing/non-numeric values in either
+   the training or the evaluation set. Next, it performs the feature selection,
+   trains a model with those features using the filtered training set and
+   generates predictions on the filtered test set.
+
+2. ``rsmpredict`` is now run to generate predictions on the evaluation set
+   using the trained model and excludes *only* those responses where any of the
+   *selected* features (a subset of the original features) has
+   missing/non-numeric values.
+
+As a result, there is a mismatch in the number of predictions generated by
+``rsmtool`` and ``rsmpredict``.
+
+|hr|
+
+.. _q_relative_betas_not_one:
+
+:fa:`quora`. **The relative betas for the linear regression model in my**
+``rsmtool`` **run do not sum to 1. Is that a problem?** [:ref:`#
```

Comment:
```suggestion
``rsmtool`` **run do not sum up to 1. Is that a problem?** [:ref:`#
```

Reply:
I am not sure this is really necessary in American English :)